contributors.json

```diff
@@ -35,6 +35,7 @@
 { "profile_name":"SimonPeatman", "organisation":"University of Reading"},
 { "profile_name":"pelson", "organisation":""},
 { "profile_name":"malcolmbrooks", "organisation":""}
+{ "profile_name":"cvelascof", "organisation":""},
```

Comment:
This is not valid JSON and will cause the about page not to render. You need a comma after every entry in a container **except** the last. Here you have no comma after the second-to-last *and* a comma after the last.